packages/cli/src/util/git-helpers.ts

```diff
@@ -37,6 +37,7 @@ export function getGitDirectory(opts: GitExecOptions): string | null {
     const gitConfigPath = execSync('git rev-parse --git-dir', {
       cwd,
       encoding: 'utf8',
+      stdio: ['pipe', 'pipe', 'ignore'],
```

Comment:
stdin can be ignored as well I think

```suggestion
      stdio: ['ignore', 'pipe', 'ignore'],
```